modules/preview-react/README.md

```diff
@@ -0,0 +1,52 @@
+# Canvas Kit Preview
+
+<img src="https://img.shields.io/badge/PREVIEW-beta-blueviolet" alt="PREVIEW: Beta" />
+
+This is a group of coming-soon components that have had a full design and a11y review, and are
+approved for use in product. Their functionality and design are set, but the API's and/or underlying
+architecture could still be subject to change. This bundle serves as a staging ground for components
+that are ready to use, but may not be up to the high code standards upheld in the main
+`@workday/canvas-kit-react` bundle.
+
+Generally, Canvas Kit Preview components were originally in [Canvas Kit Labs](../labs-react) and
+have been promoted. Essentially, Canvas Kit Labs is for alpha components and Canvas Kit Preview is
+for beta components.
```

Comment:
Are we okay with this simplification?

Reply:
I'm ok with the alpha/beta designation as a quick way to grok what Labs/Preview might mean without diving into the full documentation.